Dockerfile.windows

```diff
@@ -3,14 +3,14 @@
 # Initial stage which pulls prepares build dependencies and CLI tooling we need for our final image
 # Also used as the image in CI jobs so needs all dependencies
 ####################################################################################################
+
+ARG IMAGE_OS_VERSION=1809
+
 # had issues with official golange image for windows so I'm using plain servercore
 FROM mcr.microsoft.com/windows/servercore:ltsc2019 as builder
```

Comment:
@mweibel please also use `IMAGE_OS_VERSION` here (`1809` tag also exists for servercore). If one is using process isolation for building the image, the builder image OS version needs to match the host OS as well.